StortSpelProjekt/Game/src/Gamefiles/Components/HealthComponent.cpp

```diff
@@ -53,12 +53,12 @@ void component::HealthComponent::ChangeHealth(int hpChange)
 
 }
 
-int component::HealthComponent::GetHealth()
+const int component::HealthComponent::GetHealth() const
```

Comment:
Remove the first const keyword. We do want to be able to use the variable that we get from this.
As it is not a pointer the user will NOT be able to change the value in the class.
"int component::HealthComponent::GetHealth() const"